src/pkgs/bootstrap.nix

```diff
@@ -26,8 +26,6 @@ buildPkgs.runCommand "bootstrap" { } ''
   cp ${files.nixConf} $out/etc/nix/nix.conf
   cp ${files.resolvConf} $out/etc/resolv.conf
 
-  touch $out/etc/UNINTIALISED
```

Comment:
Wow. =D Yeah, who needs marker files when one can just write an expression that builds a script that will then later call another nix to regenerate itself. And it makes perfect sense.